src/Select/index.js

```diff
@@ -135,6 +135,10 @@ const Select = ({
     categories = allChildren.map(({ props }) => ({
       label: props.label,
       categoryChildren: React.Children.toArray(props.children),
+      // eslint-disable-next-line react/prop-types
+      kind: props.kind,
+      // eslint-disable-next-line react/prop-types
+      isFlat: props.isFlat,
```

Comment:
I'd like to address these ignores in the long term by converting this component to typescript.